packages/babel-generator/src/source-map.ts

```diff
@@ -76,24 +102,56 @@ export default class SourceMap {
     line: number,
     column: number,
     identifierName?: string | null,
+    identifierNamePos?: { line: number; column: number },
     filename?: string | null,
   ) {
     this._rawMappings = undefined;
 
+    let original;
+    let source;
+
+    if (line != null) {
+      if (this._inputMap) {
+        if (identifierNamePos) {
+          const name = originalPositionFor(this._inputMap, {
+            line,
```

Comment:
Shouldn't this be using `identifierNamePos.line`?

Reply:
Oops this is a bug.